client/my-sites/customer-home/main.jsx

```diff
@@ -522,13 +539,10 @@ class Home extends Component {
 							{ translate( 'Get all the help you need' ) }
 						</h6>
 						<div className="customer-home__card-support">
-							<img
-								src="/calypso/images/customer-home/happiness.png"
-								alt={ translate( 'Support' ) }
-							/>
+							<img src={ happinessIllustration } alt={ translate( 'Support' ) } />
 							<VerticalNav className="customer-home__card-links">
 								<VerticalNavItem
-									path="https://en.support.wordpress.com/"
+									path="https://support.wordpress.com/"
```

Comment:
The link `https://en.support.wordpress.com/` is correct here. However, it should be localized (if you want to take care of that in this PR... otherwise no change is needed on this line).

```
import { localizeUrl } from 'lib/i18n-utils';
...
path={ localizeUrl( 'https://en.support.wordpress.com' ) }
```

Reply:
Whoops, fixed, thank you!